app/services/clearancing_service.rb

```diff
@@ -0,0 +1,47 @@
+require 'csv'
+require 'ostruct'
+class ClearancingService
+
+  attr_accessor :clearancing_status
+
+  def initialize
+    self.clearancing_status = create_clearancing_status
+  end
+
+  def process_items(potential_item_ids)
```

Comment:
Should probably be `process_items!` so that the outside caller knows that modifications to the data can occur.